frontend/src/queries/query.ts

```diff
@@ -29,7 +36,6 @@ export function queryExportContext<N extends DataNode>(
 const SYNC_ONLY_QUERY_KINDS = [
     'HogQuery',
     'HogQLMetadata',
-    'EventsQuery',
```

Comment:
Was surprised that EventsQuery (aka activity view/data table) is sync-only! Please QA that async is good here

Reply:
Everything works well. I tested various nodes of `EventsQuery`.